.travis.yml

```diff
@@ -1,16 +1,12 @@
 language: android
 
-groovy:
```

Comment:
No, you're right.  I guess I thought it would contribute to the travis build matrix; `scala` in Travis does work that way, but `groovy` doesn't.

Anyways, good catch - thanks!